--- gdal-2.2.1/frmts/jpeg2000/jpeg2000_vsil_io.cpp.orig	2017-07-11 13:18:12.198404008 -0400
+++ gdal-2.2.1/frmts/jpeg2000/jpeg2000_vsil_io.cpp	2017-07-11 13:18:38.198405142 -0400
@@ -207,7 +207,7 @@
             /* The buffer must be large enough to accommodate maximum
                putback. */
             assert(bufsize > JAS_STREAM_MAXPUTBACK);
-            stream->bufbase_ = JAS_CAST(uchar *, buf);
+            stream->bufbase_ = JAS_CAST(unsigned char *, buf);
             stream->bufsize_ = bufsize - JAS_STREAM_MAXPUTBACK;
         }
     } else {
